### ISSUE TITLE
-out flag

### ISSUE THREAD

**Issue by bootstraponline** [square/certstrap#55]
An `-out` flag as exists in openssl would be nice. Currently files need to be copied manually when the same common name is used.

etcd-release has the same issue https://github.com/cloudfoundry-incubator/etcd-release/blob/master/scripts/generate-certs

```bash
# CA
certstrap --depot-path . init --passphrase '' --common-name example.com --key-bits $SIZE
mv ./example.com.key ./ca.key
mv ./example.com.crt ./ca.crt
mv ./example.com.crl ./ca.crl

# Server
certstrap --depot-path . request-cert --passphrase '' --common-name example.com --domain 'example.com'  --key-bits $SIZE
mv ./example.com.key server.key
mv ./example.com.csr server.csr

certstrap --depot-path . sign server --CA ca
openssl pkcs8 -topk8 -nocrypt -in server.key -out server.pem

# Client
certstrap --depot-path . request-cert --passphrase '' --common-name example.com --domain 'example.com'  --key-bits $SIZE
mv ./example.com.key client.key
mv ./example.com.csr client.csr
```

#54

**Comment by mrturkmen06** [comment]
Seems merged pull request #81 fixes this issue. @mcpherrinm. This issue could be overleft.

**Comment by mcpherrinm** [comment]
yup, thanks for the reminder
